### PR TITLE
Bump modules iam-role to v0.14.0 and sns-topic to v0.20.1

### PR DESCRIPTION
**Pull request by milldr**
## what
* Update the modules versions for iam-role and sns-topic

## why
* Required to resolve downstream errors with the `compliance` components

## references
* closes #41 
* closes #21

### CODE REVIEW

**Comment by milldr** [comment]
this PR includes `make github/init` with the manual fix for `.github/auto-release.yml`

**Review by nitrocode — APPROVED** [review]

**Review by nitrocode — DISMISSED** [review]

File: variables.tf [hunk 1]
```diff
@@ -35,6 +35,7 @@ variable "subscribers" {
     protocol               = string
     endpoint               = string
     endpoint_auto_confirms = bool
+    raw_message_delivery   = bool
```
Comment (Nuru) [inline variables.tf:38]: This is a breaking change. Is it really necessary?

**Review by Nuru — CHANGES_REQUESTED** [review]
We really want to avoid breaking changes if possible.

**Pull request merged**